triton/tut/gpu.rst

```diff
@@ -312,19 +312,231 @@ on efficient data loading.
 
 .. _cuda-nvprof:
 
-Profiling GPU usage with nvprof
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Profiling GPU usage with NVIDIA Profilers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NVIDIA provides a suite of profiling tools designed to help users optimize and analyze the performance of applications running on NVIDIA GPUs. These tools are essential for understanding how applications utilize GPU resources and for identifying bottlenecks in the code.
+
+Overview of NVIDIA Profilers:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Nsight Systems is a system-wide performance analysis tool that provides detailed insights into GPU workloads, CPU workloads, and their interactions. It helps identify issues related to CPU-GPU interaction, GPU utilization, memory bandwidth, and other system-level bottlenecks, such as I/O operations.
+
+Nsight Compute is a GPU kernel profiling tool specifically designed for deep-dive analysis of CUDA kernels. It provides a wide range of performance metrics.
+
+nvprof is being deprecated. It has been largely replaced by Nsight Compute and Nsight Systems, both of which provide a more comprehensive and user-friendly profiling experience.
 
-.. admonition:: NVIDIA has a new profiling suite called Nsight Compute
+Examples
+^^^^^^^^^^
 
-   ``nvprof`` has been superceded by a tool called ``ncu``, which
-   is a command line interface (CLI) for
-   `NVIDIA Nsight Compute suite <https://docs.nvidia.com/nsight-compute/index.html>`__.
-   We'll produce more up-to-date profiling examples in the future.
+We have produced some profiling examples.
 
-When using NVIDIA's GPUs you can try to use a profiling tool
-called ``nvprof`` to monitor what took most of the GPU's
-time during the code's execution.
+**To use Nsight Systems**, you need to run:
+
+.. code-block:: bash
+
+   module load cuda
+   nsys profile -o my_application_report ./my_application
```

Comment:
This makes it parse it like a console session: stuff with `$` in front is the command and other lines are output.     Not needed but might be useful sometimes.
```suggestion
.. code-block:: console

   $ module load cuda
   $ nsys profile -o my_application_report ./my_application
```